### PR TITLE
dependencies-anylysis 0.50

### PR DESCRIPTION
**Pull request by sergioboi**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -11,5 +11,5 @@ docs/check.sh
 runInBuilder "set -e;
     ./gradlew help;
     ./gradlew help -PuseCompositeBuild=false;
-    ./gradlew -p subprojects build ${GRADLE_ARGS};
+    ./gradlew -p subprojects build --scan ${GRADLE_ARGS};
```
Comment (eugene-krivobokov) [inline build.sh]: It requires to accept a license agreement from stdin.
We can avoid this by applying Gradle enterprise plugin.

File: subprojects/settings.gradle.kts [hunk 2]
```diff
@@ -142,7 +142,7 @@ pluginManagement {
                 // Can't update further
                 // https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/issues/199
```
Comment (eugene-krivobokov) [inline subprojects/settings.gradle.kts:144]: The obsolete commentary

File: build.sh (hunk 1 above)
Comment (sergioboi) [reply build.sh]: #473

**Review by eugene-krivobokov — APPROVED** [review]

File: subprojects/gradle.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # TODO move to root
 kotlin.code.style=official
-org.gradle.jvmargs=-Xmx2g -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx4g -XX:+UseGCOverheadLimit -XX:GCTimeLimit=10 -Dfile.encoding=UTF-8
```
Comment (sergioboi) [inline subprojects/gradle.properties:3]: https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/issues/216
Heap size increasing works

Comment (sergioboi) [reply subprojects/gradle.properties:3]: @dsvoronin @eugene-krivobokov fyi

File: gradle.properties [hunk 4]
```diff
@@ -1,5 +1,5 @@
 kotlin.code.style=official
-org.gradle.jvmargs=-Xmx2g -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx4g -XX:+UseGCOverheadLimit -XX:GCTimeLimit=10 -Dfile.encoding=UTF-8
```
Comment (eugene-krivobokov) [inline gradle.properties:2]: TIL
How about adding GCTimeLimit to CI build in Avito project?

**Pull request merged**